### PR TITLE
[Pending testing with vagrant env] contrail control monitor script

### PR DESCRIPTION
**Pull request by sajuptpm**
Monitor following items
* Status of XMPP connections
* Status of XMPP Peers
* Status of BGP Peers
* Status of IFMAP connection

### CODE REVIEW

File: manifests/monitoring/vpccfg.pp [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
   ## Add the name of the scripts you have added in files/monitor_scripts/
   ## Example: $monitors = ['monitor_xmpp.sh','monitor_cassandra.sh']
-  $monitors=[]
+  $monitors=['contrail_control/introspect.sh']
```
Comment (vpramo) [inline manifests/monitoring/vpccfg.pp]: you cannot create a directory , place the script in the main directory

File: manifests/monitoring/vpcctrl.pp [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
   ## Add the name of the scripts you have added in files/monitor_scripts/
   ## Example: $monitors = ['monitor_xmpp.sh','monitor_cassandra.sh']
-  $monitors=[]
+  $monitors=['contrail_control/introspect.sh']
```
Comment (vpramo) [inline manifests/monitoring/vpcctrl.pp]: Same as above

File: manifests/monitoring/vpccfg.pp (hunk 1 above)
Comment (sajuptpm) [inline manifests/monitoring/vpccfg.pp]: done

**Comment by vpramo** [comment]
Looking good ,and tested on vagrant setup

**Comment by vpramo** [comment]
@sajuptpm i tested already in vagrant , is there any testing pending or is it good to be merged.

**Pull request merged**